src/Symfony/Component/HttpKernel/EventListener/ErrorListener.php

```diff
@@ -159,16 +161,20 @@ public static function getSubscribedEvents(): array
 
     /**
      * Logs an exception.
+     * 
+     * @param ?string $logChannel
      */
-    protected function logException(\Throwable $exception, string $message, ?string $logLevel = null): void
+    protected function logException(\Throwable $exception, string $message, ?string $logLevel = null, /* ?string $logChannel = null */): void
     {
-        if (null === $this->logger) {
-            return;
-        }
+        $logChannel = (3 < \func_num_args() ? \func_get_arg(3) : null) ?? $this->resolveLogChannel($exception);
```

Comment:
```suggestion
        $logChannel = \func_get_args()[2] ?? $this->resolveLogChannel($exception);
```

Reply:
technically the suggestion is not better and doesn't match the style we use elsewhere so I'd say better ignore this one